### PR TITLE
samples: peripheral_lbs: Add Thingy:53 support

### PR DESCRIPTION
**Pull request by MarekPieta**
Change introduces support for Thingy:53.

Jira: NCSDK-10306

### CODE REVIEW

File: samples/bluetooth/peripheral_lbs/README.rst [hunk 1]
```diff
@@ -85,18 +112,18 @@ This testing procedure assumes that you are using `nRF Connect for Mobile`_.
...
 
    * ``00`` when **Button 1** is released,
    * ``01`` when **Button 1** is pressed.
 
-#. Control the color of RGB LED on the dongle or status of **LED 3** on the kit by writing the following values to the "LED" characteristic in the "Nordic LED Button Service":
+#. Control the color of RGB LED on the dongle/Thingy:53 or status of **LED 3** on the kit by writing the following values to the "LED" characteristic in the "Nordic LED Button Service":
 
-   * ``00`` to switch the LED off on the kit or turn on the red RGB LED on the dongle.
-   * ``01`` to switch the LED on on the kit or turn on the green RGB LED on the dongle.
+   * ``00`` to switch the LED off on the kit or turn off the green RGB LED on the dongle or turn off the blue RGB LED on Thingy:53.
```
Comment (pdunaj) [inline samples/bluetooth/peripheral_lbs/README.rst:123]: maybe, instead of overwriting, we should have two variants: one for dongle and another for thingy

**Review by pdunaj — APPROVED** [review]

File: samples/bluetooth/peripheral_lbs/README.rst (hunk 1 above)
Comment (MarekPieta) [reply samples/bluetooth/peripheral_lbs/README.rst:123]: Or maybe we could even call it e.g. "user controlled LED" and explain which hardware LED is used on which board in `User interface` subsection.?

Comment (MarekPieta) [reply samples/bluetooth/peripheral_lbs/README.rst:123]: @greg-fer what's your opinion on this?

**Review by greg-fer — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Comment by MarekPieta** [comment]
Fixed typo in comment

**Review by pdunaj — APPROVED** [review]

**Pull request merged**